jvm-libs/linea/blob-decompressor/src/test/kotlin/net/consensys/linea/blob/GoNativeBlobDecompressorTest.kt

```diff
@@ -79,10 +83,29 @@ class GoNativeBlobDecompressorTest {
         ),
       ),
     )
+    val tx2 = TransactionFactory.createTransaction(
```

Comment:
@fluentcrafter Are you Ok commenting out compression of this transaction instead with a TODO to enable it later? I think it's better to have this test enabled without Type 4 transaction instead